schema.json

```diff
@@ -93,6 +93,10 @@
                 "isResponseDictionary": {
                     "type": "boolean",
                     "description": "(Optional) Set to true if the batch resource returns the results as a dictionary with key mapped to values (instead of a list of items). If this option is supplied `reorderResultsByKey` should not be. Default: false"
+                },
+                "uniqueBatchKeys": {
```

Comment:
`uniqueBatchKeys` confused me a little bit. would something like `isBatchKeyASet` be better? 
and I like the description of `commaSeparatedBatchKey` a lot, so maybe we could follow that and do 
```
"description": "(Optional) Set to true if the interface of the resource takes the batch key as a set (rather than an array as is more common). Default: false" 
```

Reply:
Great suggestion. I was really struggling with what to call this option haha. I like yours better for sure.